bluebottle/projects/migrations/0014_bank_details.py

```diff
@@ -0,0 +1,278 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    depends_on = (
+        ('organizations', '0002_auto__add_field_organizationdocument_created__add_field_organizationdo'),
+    )
+
+    def forwards(self, orm):
+        # Note: Don't use "from appname.models import ModelName".
+        # Use orm.ModelName to refer to models in this application,
+        # and orm['appname.ModelName'] for models in other applications.
+        print "GKENSGKENGKFLNLNWLFNLKESNFGKLENLKEW************"
+        for organization in orm['organizations.Organization'].objects.all():
+            print "Organization", organization.name
+            # Get the project, we use a strange related_name
+            # value (organization) on the Project model that
+            # we need to work with.
+            for project in organization.organization.all():
+                print "PROJECT", project.title
+                project.account_holder_name = organization.account_holder_name
+                project.account_holder_address = organization.account_holder_address
+                project.account_holder_postal_code = organization.account_holder_postal_code
+                project.account_holder_city = organization.account_holder_city
+                project.account_holder_country = organization.account_holder_country
+                project.account_iban = organization.account_iban
+                project.account_bic = organization.account_bic
+                project.account_number = organization.account_number
+                project.account_bank_name = organization.account_bank_name
+                project.account_bank_address = organization.account_bank_address
+                project.account_bank_postal_code = organization.account_bank_postal_code
+                project.account_bank_city = organization.account_bank_city
+                project.account_bank_country = organization.account_bank_country
+                project.account_other = organization.account_other
+
+                project.save()
+            print "---"
+        print '*****'
```

Comment:
It's a free beer for every dev team member every 'print' or 'console.log' found in a PR. 
Two beers if a reviewer doesn't find them...